devices/YarpRobotLoggerDevice/thrifts/YarpRobotLoggerDeviceCommands.thrift

```diff
@@ -0,0 +1,12 @@
+/**
+ * @file YarpRobotLoggerDeviceCommands.thrift
+ * @authors Lorenzo Moretti <lorenzo.moretti@iit.it>
+ * @copyright 2024 Artificial and Mechanical Intelligence - Istituto Italiano di Tecnologia
```

Comment:
FYI not really important, but legally both (before and after) copyright assignments are wrong. Only a proper legal entity can hold copyright, and internal divisions of an existing legal entity can't . So the only logical thing we can do here is to assign the copyright to "Fondazione Istituto Italiano di Tecnologia".

Reply:
Could you directly provide the commit suggestion?

Reply:
Done in the other conversation.